packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -582,6 +590,28 @@ class VectorGraphicsCodec {
     buffer._putUint8(_maskTag);
   }
 
+  int writePattern(
+    VectorGraphicsBuffer buffer,
+    double x,
+    double y,
+    double width,
+    double height,
+    Float64List transform,
+  ) {
+    buffer._checkPhase(_CurrentSection.commands);
```

Comment:
Isn't this part of patterns?

Reply:
I realized in development that it was easier to give a pattern it's own command type, rather than having it as it's own phase, so I'm removing it from the _CurrentSection enum.